Makefile

```diff
@@ -8,7 +8,7 @@
 .PHONY: install_deps_linux install_deps_darwin install_deps_darwin-m
 
 VER_PACKAGE=github.com/drand/drand/common
-CLI_PACKAGE=github.com/drand/drand/cmd/drand-cli
+CLI_PACKAGE=github.com/drand/drand/internal/drand-cli
```

Comment:
Why do we need to specify this CLI_PACKAGE thing actually? I would have expected `cmd/drand` to be our CLI package.
Also I think our new way of doing versioning doesn't require this anymore.

Reply:
It's currently reference here, but can probably be removed:

```	go build -o drand -mod=readonly -ldflags "-X $(VER_PACKAGE).COMMIT=$(GIT_REVISION) -X $(VER_PACKAGE).BUILDDATE=$(BUILD_DATE) -X $(CLI_PACKAGE).buildDate=$(BUILD_DATE) -X $(CLI_PACKAGE).gitCommit=$(GIT_REVISION)" ./cmd/drand
```

Reply:
Ah wow actually passing the CLI_PACKAGE var there seems to control some other make magic - taking it out throws this when building:

```
go build -o drand -mod=readonly -ldflags "-X github.com/drand/drand/common.COMMIT=8f9bfd74 -X github.com/drand/drand/common.BUILDDATE=27/04/2023@11:26:09 ./cmd/drand"
no Go files in /path/to/drand
make: *** [build] Error 1
```

Reply:
At some point we should investigate if there isn't some crust remaining from before we had a version.go file and all in the Makefile.